website/vsphere.erb

```diff
@@ -25,47 +25,70 @@
           </ul>
         </li>
 
-        <li<%= sidebar_current("docs-vsphere-resource") %>>
-          <a href="#">Resources</a>
+        <li<%= sidebar_current("docs-vsphere-resource-inventory") %>>
+          <a href="#">Inventory Resources</a>
           <ul class="nav nav-visible">
-            <li<%= sidebar_current("docs-vsphere-resource-datacenter") %>>
+            <li<%= sidebar_current("docs-vsphere-resource-inventory-datacenter") %>>
               <a href="/docs/providers/vsphere/r/datacenter.html">vsphere_datacenter</a>
             </li>
-            <li<%= sidebar_current("docs-vsphere-resource-file") %>>
-              <a href="/docs/providers/vsphere/r/file.html">vsphere_file</a>
-            </li>
-            <li<%= sidebar_current("docs-vsphere-resource-folder") %>>
+            <li<%= sidebar_current("docs-vsphere-resource-inventory-folder") %>>
               <a href="/docs/providers/vsphere/r/folder.html">vsphere_folder</a>
             </li>
-            <li<%= sidebar_current("docs-vsphere-resource-host-port-group") %>>
-              <a href="/docs/providers/vsphere/r/host_port_group.html">vsphere_host_port_group</a>
-            </li>
-            <li<%= sidebar_current("docs-vsphere-resource-host-virtual-switch") %>>
-              <a href="/docs/providers/vsphere/r/host_virtual_switch.html">vsphere_host_virtual_switch</a>
-            </li>
-            <li<%= sidebar_current("docs-vsphere-resource-license") %>>
-              <a href="/docs/providers/vsphere/r/license.html">vsphere_license</a>
-            </li>
-            <li<%= sidebar_current("docs-vsphere-resource-virtual-disk") %>>
+          </ul>
+        </li>
+
+        <li<%= sidebar_current("docs-vsphere-resource-vm") %>>
+          <a href="#">VM and Template Resources</a>
```

Comment:
it might be VMware terminology - but I'd argue `Template` could be removed here?